src/foundry/client-esm/canvas/sources/point-effect-source.d.mts

```diff
@@ -0,0 +1,134 @@
+   */
+  static defaultData: PointEffectSourceMixin.PointEffectSourceData;
+
+  /**
+   * A convenience reference to the radius of the source.
+   */
+  get radius(): number;
+
+  _initialize(data: Partial<PointEffectSourceMixin.PointEffectSourceData>): void;
+
+  _initializeSoftEdges(): void;
+
+  /**
+   * Configure the parameters of the polygon that is generated for this source.
+   */
+  protected _getPolygonConfiguration(): PointSourcePolygonConfig;
+
+  _createShapes(): void;
+
+  _drawMesh(layerId: string): PIXI.Mesh | null;
+
+  _updateGeometry(): void;
+
+  /**
+   * @deprecated since v11, until v13
+   * @remarks `"The setter PointEffectSource#radius is deprecated. The radius should not be set anywhere except in PointEffectSource#_initialize."`
+   */
+  set radius(radius);
+
+  /**
+   * @deprecated since v11, until v13
+   * @remarks `"PointEffectSource#los is deprecated in favor of PointEffectSource#shape."`
+   * @privateRemarks Actual definition is get/set
+   */
+  los: PointSourcePolygon;
+}
+
+type PointEffectSourceMixin_BaseEffectSource_Static = typeof PointEffectSource & typeof BaseEffectSource;
+
+export interface PointEffectSourceMixin_BaseEffectSource_Interface
+  extends PointEffectSourceMixin_BaseEffectSource_Static {
+  new <
+    SourceData extends BaseEffectSource.BaseEffectSourceData & PointEffectSourceMixin.PointEffectSourceData,
+    SourceShape extends PointSourcePolygon,
+  >(
+    ...args: ConstructorParameters<typeof BaseEffectSource>
+  ): PointEffectSource & BaseEffectSource<SourceData, SourceShape>;
+}
+
+type PointEffectSourceMixin_RenderedEffectSource_Static = typeof PointEffectSource & typeof RenderedEffectSource;
+
+export interface PointEffectSourceMixin_RenderedEffectSource_Interface
+  extends PointEffectSourceMixin_RenderedEffectSource_Static {
+  new <
+    SourceData extends RenderedEffectSource.RenderedEffectSourceData & PointEffectSourceMixin.PointEffectSourceData,
+    SourceShape extends PointSourcePolygon,
+    RenderingLayers extends Record<
+      string,
+      RenderedEffectSource.RenderedEffectSourceLayer
+    > = RenderedEffectSource.Layers,
+  >(
+    ...args: ConstructorParameters<typeof RenderedEffectSource>
+  ): PointEffectSource & RenderedEffectSource<SourceData, SourceShape, RenderingLayers>;
+}
+
+type PointEffectSourceMixin_BaseLightSource_Static = typeof PointEffectSource & typeof BaseLightSource;
+
+export interface PointEffectSourceMixin_BaseLightSource_Interface
+  extends PointEffectSourceMixin_BaseLightSource_Static {
+  new <
+    SourceData extends BaseLightSource.LightSourceData & PointEffectSourceMixin.PointEffectSourceData,
+    SourceShape extends PointSourcePolygon,
+    RenderingLayers extends Record<
+      string,
+      RenderedEffectSource.RenderedEffectSourceLayer
+    > = RenderedEffectSource.Layers,
+  >(
+    ...args: ConstructorParameters<typeof BaseLightSource>
+  ): PointEffectSource & BaseLightSource<SourceData, SourceShape, RenderingLayers>;
+}
```

Comment:
I appreciate the irony of me saying this given that I wrote this but _man_ is this ugly. Fortunately end users will never have to see this, really.

I only did it here because I didn't want to add another file as that'd throw off the scheme we have.